lib/afinter.c

```diff
@@ -428,6 +429,12 @@ void
 afinter_message_posted(LogMessage *msg)
 {
   g_static_mutex_lock(&internal_msg_lock);
+  if (!current_internal_source)
+    {
+      log_msg_unref(msg);
```

Comment:
@juhaszviktor : shouldn't we clear (and unref the LogMessage instances) the queue when it contains some items after a reload? Otherwise we could cache some items in situations when 
 * user starts syslog-ng with an internal source connected into a logpath
 * remove the internal source from config
 * reload syslog-ng
In this case, the queue could contain some items. Of course this is a better situation than before when we just "cache" all new internal messages without consuming them. But I think we could remove the cached items here.

Reply:
Ok, I'm going to correct this

Reply:
Patch corrected